src/app/iteration/iteration-modal/iteration-modal.component.html

```diff
@@ -22,6 +22,18 @@
           </div>
         </div>
         <div class="form-group">
+          <label for="parent-iteration">Parent Iteration</label>
+          <div class="parent-iteration-container">
+            <input id="parent-iteration" class="form-control" type="text" #iterationSearch (keyup)="filterIteration($event)"
```

Comment:
this should be a typeahead + dropdown (it is only a typeahead when I ran locally).